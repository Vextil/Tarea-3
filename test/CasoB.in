InsertarCP a 1 b 2 c 2 d 2 e 3 f 2 g 2 h 2 FIN
ExtraerCP
ExtraerCP
ElementosCP 2 1
ImprimirLista 1
CrearBalanceado 1 1
ImprimirBinario 1
Linealizar 1 2
ImprimirLista 2
Encolar 1
Desencolar
Desencolar
PartirLista 2 3
ImprimirLista 2
ImprimirLista 3
Apilar a 1
Apilar b 2
Desapilar
Desapilar
Desapilar
IngresarSolicitudes chrome 1 notepad 2 xc 1 notepad 1 FIN
SoloEnUnoSolicitudes 1 2 1
CrearLista 1 chrome notepad xc FIN
PerteneceASet 1 1
Fin
