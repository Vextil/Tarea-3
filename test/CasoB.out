1>OK
2>a - 1
3>b - 2
4>OK
5>Ordenada. c d f g h 
6>OK
7>
  (0,h)
   (0,g)
 (0,f)
  (0,d)
   (0,c)
8>OK
9>Ordenada. c d f g h 
10>OK
11>
  (0,h)
   (0,g)
 (0,f)
  (0,d)
   (0,c)

12>La cola estaba vacia.
13>OK
14>Ordenada. c d 
15>Ordenada. f g h 
16>OK
17>OK
18>(2,b)
19>(1,a)
20>La pila estaba vacia. 
21>OK
22>OK
23>OK
24>SI NO SI 
25>Fin
